src/ros_tcp_endpoint/unity_ip_sender.py

```diff
@@ -1,9 +1,24 @@
+#  Copyright 2020 Unity Technologies
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
 import rospy
 import socket
-from tcp_endpoint.RosTCPClientThread import ClientThread
-from tcp_endpoint.msg import RosUnityError
+from .client import ClientThread
+from ros_tcp_endpoint.msg import RosUnityError
+
 
-class UnityTCPSender:
+class UnityIPSender:
```

Comment:
Not sure why this name needed to change... this class handles all our outgoing connections from ROS to Unity; mainly sending stuff to subscribers. (Ironically, sending IP addresses is one of the few things it doesn't do, since the handshake response is sent by the main tcp server.)

Reply:
Ah ok I was attempting to rename for clarity here but it seems I'm muddying the waters more... what's the delineation of responsibilities between this class and the RosUnityHandshakeService (currently renamed: UnityMonitor)?  Would it be more accurate to rename this class to "UnityStatusSender" and call UnityMonitor "UnityStatusReceiver?"

Reply:
Ros Publisher messages are sent using from Unity (ROSConnection.cs) to ROS's listen socket (server.py).

Ros Subscriber messages are sent from ROS (unity_tcp_sender.py) to Unity's listen socket (ROSConnection.cs).

As for UnityMonitor, I think we should just get rid of it. I have a branch where I have deleted unity_monitor and subsumed it into unity_tcp_sender, it literally just ends up taking a couple extra lines in the process_handshake method.

Reply:
![image](https://user-images.githubusercontent.com/73140792/100672656-1edd0e00-3317-11eb-8c66-8e3d881be3c8.png)